scroll-animations-1/Overview.bs

```diff
@@ -1396,15 +1237,58 @@ spec: cssom-view-1; type: dfn;
 	should it return percentage progress through that range,
 	or time progress through that range?
 
+# Appendix B: Timeline Scope # {#timeline-scoping}
+
+	ISSUE: This section should move to CSS-ANIMATIONS-2.
+
+	This appendix introduces the 'timeline-scope' property,
+	which allows declaring a timeline name’s scope on an ancestor
+	of the timeline’s defining element.
+
+# Declaring a Named Timeline’s Scope: the 'timeline-scope' property ### {#timeline-scope}
```

Comment:
OK, I think I fixed 1 by rephrasing to “declares an [=inactive timeline=] with the specified name”. Lmk if that seems to work or if it needs additional work.

3. Yes... I think this makes more sense from an OM perspective? I think it would be confusing if some references to the named timeline referred to one object and others to a different timeline that happens to behave the same.

2. I don't mind clarifying this, but where is it specified e.g. if we assume https://github.com/w3c/csswg-drafts/pull/8902 ?

Reply:
Hmm, either you labeled the answer for (2) as (3),  and (3) as (2), or I'm not following.

(1) :+1: 
(2) (_I.e. the issue about timeline-scope being "once per frame"-things_): AHEM, it's not exactly specified in 8902. :-) The idea is that deferred timelines are scroll timelines that snapshot their state at the same time normal scroll timelines would snapshot their state, except that a deferred timeline grabs the state of its currently attached timeline.

Hmm, I'm not sure how to express that in this PR. The spec currently says this:

> To avoid such [layout cycles](https://drafts.csswg.org/scroll-animations-1/#layout-cycles), animations with a [scroll progress timeline](https://drafts.csswg.org/scroll-animations-1/#scroll-progress-timelines) update their current time once during step 7.10 of the [HTML Processing Model](https://html.spec.whatwg.org/multipage/webappapis.html#processing-model-8) event loop, as step 1 of [update animations and send events](https://drafts.csswg.org/web-animations-1/#update-animations-and-send-events).

We could maybe spec something like: scopes created by timeline-scope similarly update their associated timeline once per frame?

(3) (_I.e. the issue about "setting the timeline"_): OK.

Reply:
OK, thanks for the pointer. I made a tweak to the wording in that section to try to address this. :) Check the second commit?

Reply:
👍